src/main/scala/mesosphere/marathon/api/v2/DeploymentsResource.scala

```diff
@@ -28,7 +28,10 @@ class DeploymentsResource @Inject() (
   @GET
   def running(@Context req: HttpServletRequest): Response = authenticated(req) { implicit identity =>
     val infos = result(service.listRunningDeployments())
-      .filter(_.plan.affectedApplications.exists(isAuthorized(ViewRunSpec, _)))
+      .filter(steps => (steps.plan.affectedApplications.isEmpty ||
```

Comment:
Previously, deployment plans always had apps. Now they can have apps or pods and they definitely have at least one of them, but either could be empty, so exists is automatically false. Hence, we have to check if they're empty.